src/config/constants.ts

```diff
@@ -2,8 +2,10 @@
 import { Networks } from "@stellar/stellar-sdk";
 
 const PASSWORD_MIN_LENGTH = 8;
+const PASSWORD_MAX_LENGTH = 32;
+const HASH_KEY_EXPIRATION_MS = 48 * 60 * 60 * 1000; // 48 hours
 
-export { PASSWORD_MIN_LENGTH };
+export { PASSWORD_MIN_LENGTH, PASSWORD_MAX_LENGTH, HASH_KEY_EXPIRATION_MS };
```

Comment:
just out of curiosity, why are we exporting those consts like this? 
```
export { PASSWORD_MIN_LENGTH, PASSWORD_MAX_LENGTH, HASH_KEY_EXPIRATION_MS };
```

instead of this:
```
export const PASSWORD_MIN_LENGTH = 8;
export const PASSWORD_MAX_LENGTH = 32;
export const HASH_KEY_EXPIRATION_MS = 48 * 60 * 60 * 1000; // 48 hours
```